docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,189 @@
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| prefix                | This prefix will be appended to every index created by the exporter; must not contain `_` (underscore).                                                                                     |         |
+| numberOfShards        | The number of [shards](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#_static_index_settings) used for each new record index created.                   | 3       |
+| numberOfReplicas      | The number of shard [replicas](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#dynamic-index-settings) used for each new record index created.           | 0       |
+| variableSizeThreshold | Defines a threshold for variable size. Variables exceeding this threshold are split into two properties: `FULL_VALUE` (full content, not indexed) and `VALUE` (truncated content, indexed). | 8191    |
+| shardsByIndexName     | A map where the key is the index name and the value is the number of shards, allowing you to override the default `numberOfShards` setting for specific indices.                            |         |
+| replicasByIndexName   | A map where the key is the index name and the value is the number of replicas, allowing you to override the default `numberOfReplicas` setting for specific indices.                        |         |
+
+</TabItem>
+
+<TabItem value="bulk">
+
+To avoid too many expensive requests to the Elasticsearch/Opensearch cluster, the exporter performs batch
+updates by default. The size of the batch, along with how often it should be flushed (regardless of
+size) can be controlled by configuration.
+
+| Option | Description                                                                                                                                                    | Default |
+| ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| delay  | Delay, in seconds, before force flush of the current batch. This ensures that even when we have low traffic of records, we still export every once in a while. | `5`     |
+| size   | The amount of records a batch should have before we flush the batch                                                                                            | `1000`  |
+
+With the default configuration, the exporter will aggregate records and flush them to Elasticsearch/Opensearch:
+
+1. When it has aggregated 1000 records. 2. Five seconds have elapsed since the last flush (regardless of how many records were aggregated).
```

Comment:
```suggestion
1. When it has aggregated 1000 records. 
2. Five seconds have elapsed since the last flush (regardless of how many records were aggregated).
```